test/module/irohad/ametsuchi/tx_presence_cache_test.cpp

```diff
@@ -121,10 +121,13 @@ TEST_F(TxPresenceCacheTest, BatchHashTest) {
           tx_cache_status_responses::Missing(hash3))));
   auto tx1 = std::make_shared<MockTransaction>();
   EXPECT_CALL(*tx1, hash()).WillOnce(ReturnRefOfCopy(hash1));
+  EXPECT_CALL(*tx1, reducedHash()).WillOnce(ReturnRefOfCopy(hash1));
```

Comment:
Same here, reduced payload hash is different from transaction hash.

Reply:
🤔